dbms/src/Interpreters/DDLWorker.cpp

```diff
@@ -528,7 +528,7 @@ bool DDLWorker::tryExecuteQuery(const String & query, const DDLTask & task, Exec
     {
         current_context = std::make_unique<Context>(context);
         current_context->setCurrentQueryId(""); // generate random query_id
-        executeQuery(istr, ostr, false, *current_context, nullptr);
+        executeQuery(istr, ostr, false, *current_context, nullptr, nullptr);
```

Comment:
It's strange that there was nullptr here. Can we pass `{}`?

Reply:
done